plugins/by-name/kulala/default.nix

```
@@ -0,0 +1,251 @@
+{ lib, ... }:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "kulala";
+  originalName = "kulala.nvim";
+  package = "kulala-nvim";
+
+  maintainers = [ lib.maintainers.BoneyPatel ];
+
+  settingsOptions = {
+    curl_path = defaultNullOpts.mkStr "curl" ''
+      Path to the cURL binary. Set this if cURL is installed in a non-standard location.
+    '';
+
+    display_mode =
+      defaultNullOpts.mkEnum
+        [
+          "split"
+          "float"
+        ]
+        "split"
+        ''
+          Defines how the request/response is displayed. Options are "split" or "float".
+        '';
+
+    split_direction =
+      defaultNullOpts.mkEnum
+        [
+          "vertical"
+          "horizontal"
+        ]
+        "vertical"
+        ''
+          Defines the direction of the split window. Options are "vertical" or "horizontal".
+        '';
+
+    default_view =
+      defaultNullOpts.mkEnum
+        [
+          "body"
+          "headers"
+          "headers_body"
+        ]
+        "body"
+        ''
+          Sets the default view mode for responses. Options are "body", "headers", or "headers_body".
+        '';
+
+    default_env = defaultNullOpts.mkStr "dev" ''
+      Sets the default environment for requests. Common options are "dev", "test", or "prod".
+    '';
+
+    debug = defaultNullOpts.mkBool false ''
+      Enables or disables debug mode.
+    '';
+
+    contenttypes =
+      defaultNullOpts.mkAttrsOf lib.types.anything
```

```suggestion
      defaultNullOpts.mkAttrsOf types.anything
```
Same for the other occurrences.